scripts/typescript_checks.py

```diff
@@ -173,6 +173,7 @@
     'core/templates/pages/Base.ts',
     'core/templates/pages/admin-page/admin-page.import.ts',
     'core/templates/pages/blog-admin-page/blog-admin-page.import.ts',
+    'core/templates/pages/classroom-admin-page/classroom-admin-page.import.ts',
```

Comment:
Can you write this file in typescript instead? /cc @vojtechjelinek

Reply:
Yeah, this should be typed.

Reply:
Thank you @U8NWXD @vojtechjelinek for the confirmation.
I have updated the code, please take a look.